packages/fhir-converter/src/index.js

```diff
@@ -34,3 +35,13 @@ var server = app.listen(port, function () {
   console.log(`NODE_OPTIONS=${process.env.NODE_OPTIONS}`);
   console.log("FHIR Converter listening at http://%s:%s", host, port);
 });
+
+var loadbalancerTimeout = dayjs.duration({ minutes: 15 }).asMilliseconds();
```

Comment:
using 15 mins since thats the timeout on the [ALB](https://github.com/metriport/metriport/blob/a2b1dfa4adc5dd973deacc6ce5776fa05e202e20/packages/infra/lib/api-stack/fhir-converter-service.ts#L79)